pipelines/broad/dna_seq/germline/joint_genotyping/UltimaGenomics/UltimaGenomicsJointGenotyping.wdl

```diff
@@ -34,6 +39,23 @@ workflow UltimaGenomicsJointGenotyping {
     # than a z-score of -4.5 which is a p-value of 3.4e-06, which phred-scaled is 54.69
     Float excess_het_threshold = 54.69
 
+    #inputs for threshold filtering
+    String sample_name_truth
+    File gtr_vcf
```

Comment:
Is `gtr_vcf`, etc. conventional? Why not just `truth_`?

Reply:
Note also the use of `truth_sample_name` and `call_sample_name` in the `run_comparison_pipeline.py` command line, so it might be worth just mirroring those names as well.

Reply:
And never mind, I see `gtr` elsewhere below. But in general, it might be good to mirror arguments when possible